webview/src/plots/components/ZoomedInPlot.tsx

```diff
@@ -43,15 +44,7 @@ export const ZoomedInPlot: React.FC<ZoomedInPlotProps> = ({
   openActionsMenu
 }: ZoomedInPlotProps) => {
   const zoomedInPlotRef = useRef<HTMLDivElement>(null)
-
-  useEffect(() => {
```

Comment:
Moved this `useEffect` into a hook so we could use it in both `ZoomedInPlot` and `ErrorsModal`